configuration.ts

```diff
@@ -186,10 +186,9 @@ export class Configuration {
    * @throws {FgaValidationError}
    */
   public isValid(): boolean {
-    if (!this.apiUrl) {
-      assertParamExists("Configuration", "apiScheme", this.apiScheme);
-      assertParamExists("Configuration", "apiHost", this.apiHost);
-    }
+    if (!this.apiUrl && !this.apiScheme && !this.apiHost) {
```

Comment:
Ah I overlooked that we set a default for `this.apiScheme` maybe if we remove that from the check the test will pass?

```suggestion
    if (!this.apiUrl && !this.apiHost) {
```

Reply:
We can certainly give it a try!